components/ExerciseActions/ExerciseActions.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {exercises, paths} from '../../utils/constants';
+import {
+  getSubData,
+  getUserId,
+  pushUpdateItemInArray,
+} from '../../utils/firebaseUtils';
+import {ExerciseDetailsForm, Navigation} from '../../types/CommonTypes';
+import {useNavigation, useRoute} from '@react-navigation/native';
+import ExerciseForm from '../ExerciseForm/ExerciseForm';
+import {useQuery} from 'react-query';
+
+const ExerciseActions: React.FC = () => {
+  const navigation = useNavigation<Navigation>();
+  const uid = getUserId();
+  const route = useRoute();
+  const {workoutKey, exerciseKey} = route.params;
+  const buttonTitle = exerciseKey
+    ? exercises.EDIT_EXERCISE
+    : exercises.ADD_EXERCISE;
+
+  const getExercise = async () =>
+    await (
+      await getSubData(
```

Comment:
Chaining `awaits` or `then` is bad practice, try using variables to make it better looking.

Ex:
```tsx
const subDataResponse = await getSubData()
await subDataResponse.val()
```